test-suite/output/InvalidDisjunctiveIntro.out

```diff
@@ -0,0 +1,16 @@
+The command has indeed failed with message:
+Cannot coerce to a disjunctive/conjunctive pattern.
+The command has indeed failed with message:
+Disjunctive/conjunctive introduction pattern expected.
+The command has indeed failed with message:
+Cannot coerce to a disjunctive/conjunctive pattern.
+The command has indeed failed with message:
+Cannot coerce to a disjunctive/conjunctive pattern.
+The command has indeed failed with message:
+Ltac variable H is bound to <tactic closure> which cannot be coerced to
+an introduction pattern.
+The command has indeed failed with message:
+Disjunctive/conjunctive introduction pattern expected.
+The command has indeed failed with message:
+Ltac variable H' is bound to <tactic closure> which cannot be coerced to
+an introduction pattern.
```

Comment:
FTR the output in 8.7 is:

```
The command has indeed failed with message:
Ltac variable H' is bound to a tactic which cannot be
coerced to an introduction pattern.
```

I think this change in the error message is linked to 565a9a1b5368c586e529fc9774e4cb4b81c6441b. I'm just writing this comment to ensure that this change of `a tactic` to `<tactic closure>` in the error message was intended (cc @herbelin).

Reply:
Well, it is a generic printer and I did not know what to write. For instance, the message is  used also for `idtac x y z` where we would probably not write "a tactic" if some of `x`, `y`, or `z` is bound to a tactic.

I don't know what to to. Maybe going back to an ad hoc printer for this particular case?

Reply:
I think it's fine as-is with "<tactic closure>" - consistency between error messages is nice, both from a user and code base perspective.

Reply:
Yeah, I don't think the message is problematic either. I was just making sure that the change did not happen by accident.

Reply:
@tchajed: So writing `"a tactic closure"` (with the quotes) would be ok for you?

Reply:
I actually like the current error message, with the angle brackets. It emphasizes that there's a (currently unprintable) tactic closure assigned to that variable.

Reply:
Well, so let's keep it as it is, at least for the moment.